gobblin-temporal/src/main/java/org/apache/gobblin/temporal/yarn/YarnService.java

```diff
@@ -546,13 +547,22 @@ private void addContainerLocalResources(Path destDir, Map<String, LocalResource>
 
     FileStatus[] statuses = this.fs.listStatus(destDir);
     if (statuses != null) {
+      Set<String> libJarNames = new HashSet<>(Arrays.asList(this.config.getString(GobblinYarnConfigurationKeys.YARN_APPLICATION_LIB_JAR_LIST).split(",")));
+      String containerJars = this.config.hasPath(GobblinYarnConfigurationKeys.CONTAINER_JARS_KEY) ?
+          this.config.getString(GobblinYarnConfigurationKeys.CONTAINER_JARS_KEY) : "";
```

Comment:
don't we have a `getPath` that takes a default?

let's call within a `getAppLibJarList(this.config)` utility, providing far more facile testability